src/ui/src/app/main-header/main-header.component.html

```diff
@@ -8,7 +8,7 @@
     </div>
     <div class="main-header-search">
       <form (ngSubmit)="searchCharts(searchBox)">
-        <mat-form-field class="md-input--reverse md-input--full md-input--big">
+        <mat-form-field class="mat-input--reverse mat-input--full mat-input--big">
```

Comment:
ah I see, I guess the https://www.npmjs.com/package/angular-material-prefix-updater tool I used didn't account for class modifiers!

Reply:
This is custom css classes we used so yes I guess the updater is not so magic haha